Source/CombatExtended/CombatExtended/Projectiles/BulletCE.cs

```diff
@@ -14,16 +14,18 @@ public class BulletCE : ProjectileCE
     {
         private void LogImpact(Thing hitThing, out LogEntry_DamageResult logEntry)
         {
+	    var ed = equipmentDef;
+	    if (ed==null) ed = ThingDef.Named("Gun_Autopistol");
```

Comment:
Can be simplified to (and then inlined)
```suggestion
	    var ed = equipmentDef ?? ThingDef.Named("Gun_Autopistol");
```